src/main.js

```diff
@@ -127,28 +127,9 @@ const killProcessId = doomedProcessId => {
 };
 
 const killAllRunningProcesses = () => {
-  const processKillingPromises = processIds.map(
-    processId =>
-      new Promise((resolve, reject) => {
-        killProcessId(processId);
-
-        psTree(processId, (err, children) => {
-          if (err) {
-            return reject(err);
-          }
-
-          if (!children || children.length === 0) {
-            return resolve();
-          }
-
-          children.forEach(child => killProcessId(child.PID));
-
-          resolve();
-        });
-      })
-  );
-
-  return Promise.all(processKillingPromises).catch(err => {
+  try {
```

Comment:
@superhawk610 Is this OK? I think the child killing first is needed here for Linux/Mac. Also it looks like there is no Windows version in `killProccessId(processId)`. 
What do you think should we add `killProcess` to `platform.service`? I think this would be useful so we can use it on app exit and for task killing on dev server close.

Reply:
Great catch, I'm new to Electron so I just figured `addProcessId` and `removeProcessId` were internal Electron calls, I didn't realize we implemented them.

Yeah, we should probably move this to `platform.service` to avoid duplicating the implementation in `main.js` and `task.middleware`. I'll take this.

Reply:
Moved it into `services/kill-process-id.service.js` so it can be loaded both server- and client-side.